certs/manager.go

```diff
@@ -488,10 +501,18 @@ func (c *CertificateManager) Add(certData []byte, orgID string) (string, error)
 		return "", err
 	}
 
+	c.storage.AppendToSet(orgID+"-index", "raw-"+certID)
+
 	return certID, nil
 }
 
 func (c *CertificateManager) Delete(certID string) {
+
+	if len(certID) >= sha256.Size*2 {
```

Comment:
Can we remove this somehow? Maybe move this logic from CertificateManager to Certificate API handler level, where we can have access to the orgID?